specs/pvs/corla.pvs

```diff
@@ -13,15 +13,31 @@
 %%% risk limiting audits across all counties in the state
 %%% simultaneously.
 %%%
+%%% \subsection{Colorado Statues and Rules}
+%%%
 %%% The Colorado election law pertaining to election audits is
 %%% \href{http://law.justia.com/codes/colorado/2016/title-1/general-primary-recall-and-congressional-vacancy-elections/article-7/part-5/section-1-7-515/}
-%%% {Section 1-7-515, C.R.S.}, where ``C.R.S.'' denotes ``Colorado Revised Statutes''. The rules pertaining to election audits are
-%%% found in the ``Revised Draft of Proposed Rules'' whose latest
-%%% version is labeled CCR 1505-1 and dated July 6, 2017. Section 25,
-%%% Post-Election Audit, of that document is the salient portion of
-%%% that document for our purposes. We will call that document
-%%% RDPR-6-Jul-2017 for the purpose of traceability in this
-%%% specification.
+%%% {Section 1-7-515, C.R.S.}, where ``C.R.S.'' denotes ``Colorado Revised Statutes''.
+%%%
+%%% The rules pertaining to election audits are found in Code of
+%%% Colorado Regulations, Secretary of State, State of Colorado,
+%%% Department of State Elections Rules
+%%% \href{http://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=7121&fileName=8%20CCR%201505-1}{\textbf{8
+%%% CCR 1505-1}}. We will call that document \emph{Rules 1505-1} for
```

Comment:
That version is not yet updated.
For today at least, Rule 25 is here:
http://www.sos.state.co.us/pubs/rule_making/CurrentRules/8CCR1505-1/Rule25.pdf
and others are at http://www.sos.state.co.us/pubs/rule_making/CurrentRules/8CCR1505-1Elections.html